src/whitenoise/asgi.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import asyncio
+
+from asgiref.compatibility import guarantee_single_callable
+
+from .string_utils import decode_path_info
+from whitenoise.base import BaseWhiteNoise
+
+# This is the same block size as wsgiref.FileWrapper
+DEFAULT_BLOCK_SIZE = 8192
+
+
+class AsgiWhiteNoise(BaseWhiteNoise):
+    def __init__(self, *args, **kwargs):
+        """Takes all the same arguments as WhiteNoise, but also adds `block_size`"""
+        self.block_size = kwargs.pop("block_size", DEFAULT_BLOCK_SIZE)
```

Comment:
I am debating whether we should actually allow the user to customize this block size.

Mainly because right now it feels a bit disjointed that the WSGI block size can't be modified, but ASGI block sizes can.